app/views/DeploymentHistoryPage.scala.html

```diff
@@ -130,6 +134,12 @@ <h1>Deployments</h1>
 
 var searchBox = document.getElementById("search");
 
+serviceList.on('searchComplete', function(){
+  var numberOfDeployments = serviceList.update().matchingItems.length;
```

Comment:
What does the `update()` here do?
What happens if you just have `serviceList.matchingItems.length`?

The update may indeed be required; I'd just like to understand the behavior.